R/gtrends.R

```diff
@@ -180,6 +180,7 @@ gtrends <- function(
 #'   used.
 #'   
 #' @import ggplot2
+#' @importFrom ggplot2 ggplot labs theme_bw theme geom_line
```

Comment:
This is not needed since there is line 83

Reply:
Yes, that is true. I think it just makes the package lighter and a bit easier to maintain, but this is beyond the isue.

Reply:
Can you remove line 182?

Reply:
Yes, I did.  But be careful with the #164 issue. If it is not resolved, it can also make ggplot2 fail, for example if a character "@" in gets into the query.